server/service/apple_mdm.go

```diff
@@ -1154,45 +1153,6 @@ func (svc *Service) ListMDMAppleDevices(ctx context.Context) ([]fleet.MDMAppleDe
 	return svc.ds.MDMAppleListDevices(ctx)
 }
 
-type listMDMAppleDEPDevicesRequest struct{}
```

Comment:
Curious: why were these removed?

Reply:
not used! I guess something from the initial PoC for MDM